MultiplexReservationSystem/src/main/java/p/lodz/pl/multiplexreservationsystem/model/TicketType.java

```diff
@@ -0,0 +1,14 @@
+package p.lodz.pl.multiplexreservationsystem.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum TicketType {
+  ADULT(25),
+  STUDENT(18),
+  CHILD(12.5);
+
+  private final double price;
```

Comment:
Czy hajsy w double to dobry pomysł i dlaczego nie?

Reply:
double/float traci cyfry znaczące konsekwencja w wyniku działań arytmetycznych. Powodem jest zapis liczb rzeczywistych w pamięci. Poprawnym typem byłby np. java.util.Currency

Reply:
A jak przechowywałbyś **kwotę** pieniędzy?